lib/iex/test/test_helper.exs

```diff
@@ -103,3 +105,32 @@ defmodule IEx.Case do
     |> String.trim()
   end
 end
+
+PathHelpers.write_beam(
```

Comment:
Create a `write_beam` specific to IEx and have it already purge and delete the modules for you :)

Reply:
Done. But I put it into a module since I need it in some specific tests.